src/components/presentational/Chat/Chat.tsx

```diff
@@ -84,6 +86,13 @@ export default function (props: ChatProps) {
                     </div>
                 </div>
             </div>
+            {props.suggestions && props.onSuggestionSelected && (
+                <div className="mdhui-chat-suggestions">
```

Comment:
When the view port is small mobile or large tablet the presentation of the suggestions looks strangle side by side. Changing the flex-direction on .mdhui-chat-suggestions = column, imho, makes it more visually appealing

Reply:
Screenshot? Looks reasonable to me. Was also following the chatGPT model they have things lined up the same way.

<img src="https://github.com/user-attachments/assets/fc879e2e-51ec-4eb5-a41b-9a2019f7f6c2" width="400" />

Reply:
@vladtar Thanks for drawing my attention to chatgpt on mobile.  Chatgpt will display wider text and allows the scrolling left/right through suggestions. The multiple lines of the current format are what stand out as clunky. 
There is also a neat approach to suggestions within ChatGpt where a generalized question is in bold - What is my average - for example - and below in sub text is a more direct phrase like, sleep each night/heart rate this week/steps per day?

Reply:
![image](https://github.com/user-attachments/assets/d64af9cd-5f54-4062-8ddb-3814e4c5c74b)